picosystem/src/display.rs

```diff
@@ -30,17 +30,28 @@ pub fn framebuffer() -> &'static mut [u16; WIDTH * HEIGHT] {
     unsafe { &mut FRAMEBUFFER }
 }
 
-pub type RealDisplay =
-    ST7789<SPIInterfaceNoCS<Spi<hal::spi::Enabled, pac::SPI0, 8>, DynPin>, DynPin>;
+pub type RealDisplay = st7789::ST7789<SPIInterfaceNoCS<Spi<hal::spi::Enabled, pac::SPI0, 8>, DynPin>, DynPin, DynPin>;
 
 pub struct Display {
     st7789: RealDisplay,
-    backlight_pin: DynPin,
     lcd_vsync_pin: DynPin,
     dma_channel: DmaChannel,
     last_vsync_time: u32,
 }
 
+
```

Comment:
Dead code

Reply:
Well, if you say so, I would suggest we remove it.